spray-json-shapeless/src/main/scala/org/ensime/json/FamilyFormats.scala

```diff
@@ -0,0 +1,391 @@
+package org.ensime.json
```

Comment:
shapeless-contrib is already too much of a grab-bag of random stuff ... it'd be better if it was broken out into a standalone project.